recipes/eccodes/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.13.0" %}
+
+package:
+  name: eccodes
+  version: {{ version }}
+
+source:
+  url: https://software.ecmwf.int/wiki/download/attachments/45757960/eccodes-{{ version }}-Source.tar.gz?api=v2
+  fn: eccodes-{{ version }}-Source.tar.gz
+  md5: 46a0a520f647cfa2fb5f287f22269603
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - cmake
+    - boost
+    - jasper
+    - libpng
+    - libnetcdf
```

Comment:
Can you change these to the pinned versions. (See https://github.com/conda-forge/staged-recipes/wiki/Pinned-dependencies)

```yaml
- libpng 1.6*
- libnetcdf 4.3*
```

You will also need to change them below.